en/services/multi_gcs.md

```diff
@@ -0,0 +1,128 @@
+# Multi GCS protocol
```

Comment:
Can you make your repo accept change requests from master? That would allow me to subedit freely (ideally) and merge comments. Likely to be much faster and less work on you.